source/install/install-tar.rst

```diff
@@ -88,6 +88,21 @@ You may also remove the Mattermost systemd unit file and the user/group created
 Frequently asked questions
 --------------------------
 
+Can I run Mattermost without a proxy?
```

Comment:
I would place this after the FAQs about starting at boot and any mentions of nginx.